### DISCUSSION TITLE
Network 192.168.56.10 is not available.

### DISCUSSION THREAD

**Discussion by alevikpes** [UtrechtUniversity/yoda#479] [Q&A] [answered]
Hi!

I am trying to install yoda with irods and following your documentation it must be as easy as running `vagrant up`. This didn't work for me. I cloned the repo, switched to development branch and ran `vagrant up`. I got this error `Network 192.168.56.10 is not available. Specify available network name, or an ip address if you want to create a new network.`

I am not familiar with vagrant or any VM and I only understand that something related to the IPs must be configured in vagrant.

When I grep this IP in the repo directory, it shows me a couple of dozens of files, where this IP is hardcoded.

Can you please help me with this or point me to some page, where it is explained?

Thank you.

PS. There are also `release` branches. Should I use those?

**Comment by stsnel** [comment]
Hi,

I have a few questions:
- What operating system are you using?
- What Vagrant provider are you trying to use?
- If you look at your network configuration, is the 192.168.56.0/24 subnet already in use for something else?

Using the development branch is fine for trying out Yoda. If you want to use Yoda in a production / PoC like setting, we recommend using a tagged version (e.g "v1.9.4").

Best,
Sietse

**Comment by alevikpes** [comment]
I am currently on Fedora 39 just testing, but eventually, it will be running on a server with RHEL9.

I don't know what you mean by `Vagrant provider`. I have VirtualBox, if this is what you mean.

I don't think this subnet is in use.

Just came to my mind: Do I need to run `vagrant up` inside the VM or on the host system?

**Comment by stsnel** [comment]
Right, I was referring to the software that Vagrant uses to manage VMs (in this case Virtualbox). Do you have a `/etc/vbox/networks.conf` configuration file? If so, which subnets are listed there?

You indeed need to run `vagrant up` on the host system rather than in the VM.

**Comment by alevikpes** [comment]
No, the directory `/etc/vbox/` doesn't exist.

**Reply by stsnel** [reply]
Okay, if you open the VirtualBox GUI and go to `File -> Tools -> Network manager`, what host-only networks do you see?

**Comment by alevikpes** [comment]
There are no networks

**Reply by stsnel** [reply]
If you add a network with prefix 192.168.56.1/24 (netmask 255.255.255.0) in VirtualBox and retry the `vagrant up` command, do you still get the same error message? If not, can you share the output of `vagrant up --debug`?

**Reply by alevikpes** [reply]
It is still the same error. Here is the output in the file.
[vagrant-debug.txt](https://github.com/user-attachments/files/17324381/vagrant-debug.txt)

**Reply by stsnel** [reply, marked as answer]
I see. Does it work if you change

```
ENV['VAGRANT_DEFAULT_PROVIDER'] = "libvirt"
```

in the Vagrantfile to

```
ENV['VAGRANT_DEFAULT_PROVIDER'] = "virtualbox"
```
? If not, what is the `vagrant up` debug output in that case?

**Reply by alevikpes** [reply]
It looks like this was the problem. I should have also downgraded the virtualbox to v 7.0, since it was incompatible with the vagrant version.

Any way, there is another error while booting the VM:
```bash
Bringing machine 'combined' up with 'virtualbox' provider...
==> combined: Box 'generic/ubuntu2004' could not be found. Attempting to find and install...
    combined: Box Provider: virtualbox
    combined: Box Version: >= 0
==> combined: Loading metadata for box 'generic/ubuntu2004'
    combined: URL: https://vagrantcloud.com/generic/ubuntu2004
==> combined: Adding box 'generic/ubuntu2004' (v4.3.12) for provider: virtualbox
    combined: Downloading: https://vagrantcloud.com/generic/boxes/ubuntu2004/versions/4.3.12/providers/virtualbox/amd64/vagrant.box
    combined: Calculating and comparing box checksum...
==> combined: Successfully added box 'generic/ubuntu2004' (v4.3.12) for 'virtualbox'!
==> combined: Importing base box 'generic/ubuntu2004'...
==> combined: Matching MAC address for NAT networking...
==> combined: Checking if box 'generic/ubuntu2004' version '4.3.12' is up to date...
==> combined: Setting the name of the VM: combined
==> combined: Clearing any previously set network interfaces...
==> combined: Preparing network interfaces based on configuration...
    combined: Adapter 1: nat
    combined: Adapter 2: hostonly
==> combined: Forwarding ports...
    combined: 22 (guest) => 2222 (host) (adapter 1)
==> combined: Running 'pre-boot' VM customizations...
==> combined: Booting VM...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "0d3d8943-55e5-400f-853f-348a3187e08c", "--type", "headless"]

Stderr: VBoxManage: error: The virtual machine 'combined' has terminated unexpectedly during startup with exit code 1 (0x1)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component MachineWrap, interface IMachine
```

**Reply by alevikpes** [reply]
Ignore my previous post - it was some installation problem. It is fixed and `vagrant up` worked.

**Reply by alevikpes** [reply]
Thank you for your help. Will continue with the guide.

**Reply by stsnel** [reply]
Thanks for letting us know :+1: , we'll look into updating the docs for this scenario
